package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@mapbox/shp-write",
-  "version": "0.4.1",
+  "version": "0.4.2",
   "description": "write shapefiles from pure javascript",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "test": "mocha -R spec",
     "prepublish": "npm run make",
-    "make": "browserify -s shpwrite ./ > shpwrite.js"
+    "make": "browserify -s shpwrite ./ > dist/index.min.js"
```

Comment:
Should we just remove this step? It does seem we need a unified .js file for the unpkg link